pkg/importer/dataStream_test.go

```diff
@@ -1,20 +1,41 @@
 package importer
 
 import (
+	"archive/tar"
+	"bufio"
+	"bytes"
+	"encoding/hex"
 	"io"
-	"net/url"
+	"io/ioutil"
+	"math/rand"
+	"os"
+	"path/filepath"
 	"reflect"
+	"strings"
 	"testing"
 
+	"github.com/xi2/xz"
 	"kubevirt.io/containerized-data-importer/pkg/image"
-	"path/filepath"
 )
 
+func createSimpleDataStream(ep, accKey, secKey string) *dataStream {
```

Comment:
why this func and `createDataStream` when they appear to be the same?

Reply:
ack